draft-irtf-t2trg-rest-iot.md

```diff
@@ -520,13 +547,19 @@ A layered system is easier to modify, as topology changes become transparent.
 Furthermore, this helps scalability, as intermediaries such as load balancers can be introduced without changing the client side.
 The clean separation of concerns helps with simplicity.
 
+IoT systems greatly benefit from this constraint, as it allows to effectively shield constrained devices behind intermediaries and is also the basis for gateways, which are used to integrate other (IoT) ecosystems.
+
 ## Code-on-Demand
 
 This principle enables origin servers to ship code to clients.
 
 Code-on-Demand improves modifiability, since new features can be deployed during runtime (e.g., support for a new representation format).
 It also improves performance, as the server can provide code for local pre-processing before transferring the data.
 
+In IoT systems, code-on-demand has limited use in its traditional form.
```

Comment:
s/has/has seen/ ?
While there's limited use as of today, I'm not sure we have explored these options to the fullest yet, so would consider toning this down a little.

Reply:
Yes, let's leave this open. Who knows what will be...